src/core_plugins/kibana/server/routes/api/export/index.js

```diff
@@ -0,0 +1,22 @@
+import exportDashboards from '../../../lib/export/export_dashboards';
+import Boom from 'boom';
+import moment from 'moment';
+export default function exportApi(server) {
+  server.route({
+    path: '/api/kibana/export/dashboards',
```

Comment:
Any reason for not inverting the path? Unless we are going to add more importers and exporters `/api/kibana/dashboards/export` and `/api/kibana/dashboards/import` makes more sense to me.